lib/commands/report/create.js

```diff
@@ -181,7 +181,17 @@ async function setupCommand (name, description, argv, importMeta) {
       }
     })
 
-  const packagePaths = await getPackageFiles(cwd, cli.input, config, debugLog)
+  // TODO: setupSdk(getDefaultKey() || FREE_API_KEY)
```

Comment:
since the login is supported can we do this now?

Reply:
Gotta wait until #46 is in unfortunately